src/main/java/com/xceptance/neodymium/recording/config/GifRecordingConfigurations.java

```diff
@@ -0,0 +1,18 @@
+package com.xceptance.neodymium.recording.config;
+
+import org.aeonbits.owner.Config.Sources;
+
+@Sources(
+{
+  "${recording.temporaryConfigFile}", "system:env", "system:properties", "file:config/dev-gif-recording.properties", "file:config/gif-recording.properties"
```

Comment:
We should add neodymium.properties and dev-neodymium properties here, to get a better coverage, with less confusion, which property needs to be where